src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+  color: $main-text-color;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: $card-main-color;
+  transition: transform 0.3s;
+
+  &__title {
+    display: flex;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+    margin-bottom: 4px;
+    transition: color 0.3s;
+  }
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The transform scale value of 1.2 on hover might be too large, causing layout shifts that could affect user experience. Consider using a smaller scale value to minimize this effect.